kernel/arch/i386/gdt.S

```diff
@@ -0,0 +1,59 @@
+#*************************************************
+#	gdt.inc
+#		-GDT Routines
+#
+#	OS Development Series
+#*************************************************
+ 
+#%ifndef __GDT_INC_67343546FDCC56AAB872_INCLUDED__
```

Comment:
co te definey oznaczają i czemu są zakomentowane
jeśli chcesz użyć define'a to: %define -> .define

Reply:
.define wywala błąd :/ "arch/i386/gdt.S:10: Error: unknown pseudo-op: `.define'"

Zakomentuję to na przyszłość na wypadek, gdyby był jakiś problem.